src/test/java/Tree/BinaryTreeTest.java

```diff
@@ -76,7 +78,20 @@ public void testpostOrder() {
                 new Node(4));
 
         List<Node> actual = BinaryTree.postOrder(root);
-        Assert.assertTrue(actual.size() == 7);
-        Assert.assertEquals(actual, expected);
+        assertTrue(actual.size() == 7);
+        assertEquals(actual, expected);
     }
+
+    @Test
+    public void testSearch() {
+
+        Node root = new Node(4);
+        BinarySearchTree.add(root, 2);
+        BinarySearchTree.add(root, 6);
+        BinarySearchTree.add(root, 1);
+
+
+        Node r = BinarySearchTree.search(root, 1);
+        assertEquals(1, r.getValue()); // 1

```

Comment:
You still just have a single test case here: you need to ensure, at a minimum, that you get back a null result for a value not in the tree.